packages/node_modules/cerebral/src/views/preact/StateContainer.js

```diff
@@ -8,7 +8,7 @@ class StateContainer extends Component {
     }
   }
   render() {
-    return this.props.children
+    return this.props.children[0]
```

Comment:
Does this mean it is fixed? :) `StateContainer` is legacy stuff btw, `UniversalController` is really the replacement